docs/models/c2hdm.rst

```diff
@@ -0,0 +1,37 @@
+.. _c2hdm:
+
+C2HDM
+==============
+
+This implementation was based on [`2108.03580 <https://arxiv.org/abs/2108.03580>`_]. 
+
+On the scalar sector there are two doublets 
+
+.. math::
```

Comment:
Is it possible to use the doxygen math here? Just to not have to type this twice for future models.
But not really a problem if it does not work